documents/social_contract.md

```diff
@@ -15,7 +15,9 @@ The communication in the group is done through Slack and through the meetings of
 
 ## Deadlines
 
-All turn-ins (Group and individual) is turned in on Thursdays. The SCRUM master will make sure that the members have turned in their parts.
+All turn-ins (Group and individual) is turned in on Thursdays.
```

Comment:
Should this not be on mondays/sundays?

Reply:
It felt like we had yet to decide as a group whether or not we should move the individual reflection to mondays as well.

Reply:
I felt like we agreed on doing both the group and individual discussion on Mondays